survey_form/index.html

```diff
@@ -45,6 +45,35 @@
 
     <form class="form">
       <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+
+      <div class="form__item">
+        <label for="name">Name: </label>
+        <input
+          type="text"
+          id="name"
+          class="form__item--input"
+          required
+          placeholder="Enter your name"
```

Comment:
W placeholderach jedną z praktyk jest podawanie przykładu poprawnej wartości dla danego inputa, np. "John Doe". W tym momencie nie duplikujemy tekstu z labela. Także możesz to zmienić w poniższych przykładach.